### ISSUE TITLE
Is it possible to unpin an element which has already been pinned?

### ISSUE THREAD

**Issue by algogrit** [webpop/jquery.pin#20]
I am trying to design an interaction where the user can decide to pin a given element in the UI. And I want to design it to toggleable, is there any way for me to 'unpin' that element, short of reattaching the element to Dom?

**Comment by jssoscar** [comment]
Yeah,when DOM ready,get the offset top of the pin element,set as the minScrollRange.Then bind scroll event handler for the Window,if the Window offset top less than the minScrollRange,then unpin the element.And you can define the maxScrollRange.

**Comment by algogrit** [comment]
I guess there's some confusion.. The way we have a pin function in jQuery is there a clean approach similar where I need to call the 'unpin' method. I could not find any such feature in documentation or even the source code.

**Comment by jssoscar** [comment]
You can define two function : pin,unpin.When window scroll,call the pin or unpin function just OK.

**Comment by algogrit** [comment]
I see the pin function https://github.com/webpop/jquery.pin/blob/gh-pages/jquery.pin.js#L3. Where is the unpin function you are referring to?

**Comment by jssoscar** [comment]
I never use the jquery.pin.js...I write by myself.

**Comment by ksweetie** [comment]
If anyone stumbles onto this later with the same problem... I found a hacky solution to do this:

```
$('.pinned').attr('style', '').removeData('pin');
$('.pinned').pin();
$(window).scroll();
```

Seems to be working for me. Triggering the scroll was because it wasn't taking effect until a scroll for some reason. /shrug
